src/analyze.ts

```diff
@@ -151,6 +153,17 @@ export const extractImports = (root: t.File) => {
   return varToImport;
 };
 
+export const getHeadings = (root: any) => {
+  const headings = [] as string[];
+  ['h1', 'h2', 'h3', 'h4'].forEach((tag) => {
```

Comment:
Would it be difficult to have options for that? People probably have different opinions about "what should be searchable" and maybe want something like

```
docs: {
  includeInSearch: 'h1, h2'
}
```

or something?

Reply:
Good idea. I'll implement it all end-to-end using this simple method and then see how disruptive it is to make it configurable